ubtue_Brill.js

```diff
@@ -44,15 +43,17 @@ function detectWeb(doc, url) {
 
 function getSearchResults(doc) {
 	let items = {};
-    let found = false;
-    let links = ZU.xpath(doc, '//a[contains(@class, "c-Typography--title")]');
-	let text = ZU.xpath(doc, '//a[contains(@class, "c-Typography--title")]/span')
-	for (let i = 0; i < links.length; ++i) {
-		let href = links[i].href;
-		let title = ZU.trimInternal(text[i].textContent);
-        if (!href || !title) continue;
-        if (!href.match(/article-.+\.xml$/))
-            continue;
+	let found = false;
+	var results = ZU.xpath(doc, '//a[contains(@class, "c-Typography--title")]');
+	if (results.length<1) {
+		results = ZU.xpath(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "flex-row", " " )) and contains(concat( " ", @class, " " ), concat( " ", "flex-nowrap", " " ))]//*[contains(concat( " ", @class, " " ), concat( " ", "c-Button--link", " " ))]');
+	}
+	for (var i in results) {
+		let href = results[i].href;
+		let title = ZU.trimInternal(results[i].textContent);
+		if (!href || !title) continue;
+		if (!href.match(/article-.+\.xml$/))
+			continue;
```

Comment:
This seems to break the previous logic. The new approach should only be used if the original one fails. i.e. `items` is empty after the extraction. Moreover from the example in https://brill.com/view/journals/bi/28/3/bi.28.issue-3.xml it is probably sufficient to use `ZU.xpath(doc,'//a[@class="c-Button--link"]'` and then filter by `href.match(/article-.+\.xml$/` to get relevant links.

Reply:
I simplified the code. For the zotaut rss procedure we don't need multiple download.